cli/cli_app.go

```diff
@@ -42,6 +42,24 @@ var DEPRECATED_COMMANDS = map[string]string{
 	CMD_TEAR_DOWN: CMD_DESTROY_ALL,
 }
 
+var TERRAFORM_COMMANDS_THAT_USE_STATE = []string{
+	"apply",
+	"destroy",
+	"env",
+	"import",
+	"graph",
+	"output",
+	"plan",
+	"push",
+	"refresh",
+	"show",
+	"taint",
+	"untaint",
+	"validate",
+	"force-unlock",
+	"state",
+}
```

Comment:
This change is not directly related to this PR, but also something I missed when updating Terragrunt to work with Terraform 0.9.